app/__init__.py

```diff
@@ -1,24 +1,18 @@
+import urllib.parse
 from flask import Flask
 from flask_babel import Babel
 
-try:
-    import urllib.parse
-    quote = urllib.parse.quote
-except:
-    import urllib
-    quote = urllib.quote_plus
-
 from config import constants
 
-class MyFlask(Flask):
+class OriginWeb(Flask):
     def get_send_file_max_age(self, name):
         if name.startswith('js/') or name.startswith('css/'):
             return 0
-        return super(MyFlask, self).get_send_file_max_age(name)
+        return super(OriginWeb, self).get_send_file_max_age(name)
 
-app = MyFlask(__name__,
+app = OriginWeb(__name__,
     template_folder=constants.TEMPLATE_ROOT,
     static_folder=constants.STATIC_ROOT)
 
 # `.encode('utf8')` will not be needed for python 3
-app.jinja_env.filters['quote_plus'] = lambda u: quote(u.encode('utf8'))
+app.jinja_env.filters['quote_plus'] = lambda u: urllib.parse.quote(u.encode('utf8'))
```

Comment:
Comment above seems to say encode is not needed in python3. Which is true?

Reply:
Thanks for pointing out the comment.  I glossed over it.

[According to the docs](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.quote):

> string may be either a str or a bytes object.

So, will remove the `encode()` method call for clarity since I'm going to change this file anyway to remove the now-irrelevant comment.

Reply:
Also, the `quote_plus` templatetag isn't actually used anywhere currently.